pmm-app/src/pmm-qan/panel/provider/provider.tools.ts

```diff
@@ -91,6 +96,7 @@ export const parseURL = (query) => ({
   querySelected: !!query.get('filter_by') || query.get('query_selected') === 'true',
   groupBy: query.get('group_by') || 'queryid',
   openDetailsTab: query.get('details_tab') || 'details',
+  dimensionSearchText: query.get('dimensionSearchText') || ''
```

Comment:
I think here there should be a nullish coalescing operator instead of `||`, also for the previous lines

Reply:
I can change for dimensionSearchText because I know it can't be null. But for the others I'm not sure if null is valid for the implementation

Reply:
IMHO `||` is still good in a number of cases. `??` should come in play where `||` is not good enough. I prefer to be explicit.